regex-assembly/942380.ra

```diff
@@ -3,23 +3,16 @@
 
 ##!+ i
 
-\bhaving\b\s+\d{1,10}\s*?[=<>]
-\bhaving\b\s+'[^=]{1,10}'\s*?[=<>]
-\bexecute\(
-\bexecute\s{1,5}[\w\.$]{1,5}\s{0,3}
-\bhaving\b ?\d{1,10} ?[=<>]+
-\bhaving\b ?[\'\"][^=]{1,10}[\'\" ?[=<>]+
-\bcreate\s+?table.{0,20}?\(
+\bhaving\b\s*\(?\s*(?:['"]?\s*\d+\s*['"]?|['"]\s*[^\d]*\s*['"]|true|false)\s*(?:[=<>+\-*/%^&!|]+|div|mod)
+\bexecute\s*\(
+\bexecute\s*`?\s*[\w\.$]+\s*`?
+\bcreate\s+table\s*`?\s*.+?\s*`?\s*\(
 \blike\W*?char\W*?\(
-select.*?case
-from.*?limit
-order\sby
-exists\s\sselect
-exists\sselect\Sif\s\(
-exists\sselect\Sifnull\s\(
-exists\sselect\Stop
-exists\sselect\Sconcat
-exists\ssystem\s\(
-exists\s\bhaving\b\s+\d{1,10}
-exists\s'[^=]{1,10}'
-\bexists\s*?\(\s*?select\b
+\bselect.*?case
+\bfrom.*?limit
+\border\s+by
+\bexists\s+select\b
+\bexists\s+system\s*\(
+\bexists\s+\bhaving\b\s+(?:\d+|true)
+\bexists\s+['"][^=]+['"]
+\bexists\s*\(\s*select\b
```

Comment:
it doesn't help if we add `\b` at the end of each word?

Reply:
What exactly do you mean? I don't get you.

Reply:
```
\bselect.*?\bcase\b
\bfrom.*?\blimit\b
\border\s+\bby\b
\bexists\b\s+\bselect\b
\bexists\b\s+\bsystem\s*\(
\bexists\b\s+\bhaving\b\s+(?:\d+|true)
\bexists\b\s+['"][^=]+['"]
\bexists\b\s*\(\s*\bselect\b
```

Reply:
Why?

Reply:
Why what? Does it help if you add that or not?

Reply:
@fzipi Our `crs-toolchain` automatically adds a `\x0b` for us.

Reply:
azurit, you didn't explain why adding `\b` without `\s` is okay. Looks like you added it per fzipi's suggestion.